### PR TITLE
Time#days_in_year returns the no of days in a given year

### PR DESCRIPTION
**Pull request by coderhs**
In reference to issue #5157

### CODE REVIEW

**Comment by sdogruyol** [comment]
LGTM :+1: ?

File: src/time.cr [hunk 1]
```diff
@@ -398,6 +398,11 @@ struct Time
     days[month]
   end
 
+  # Returns how many days are there in a year
```
Comment (luislavena) [inline src/time.cr]: Can we change description/comment like this?

```
# Returns number of days in *year*
#
# ```
# Time.days_in_year(1990) # => 365
# Time.days_in_year(2004) # => 366
# ```
```

That way, `year` is clearly identified as argument and the example provides a nice touch 😄 

Thank you! ❤️ ❤️ ❤️ 

Ref: https://crystal-lang.org/docs/conventions/documenting_code.html

Comment (RX14) [reply src/time.cr]: There should be a `.` at the end of the doc sentence.

File: src/time.cr [hunk 2]
```diff
@@ -398,6 +398,11 @@ struct Time
...
+  def self.days_in_year(year : Int) : Int32
+    days_in_month(year, 2) + 337
```
Comment (straight-shoota) [inline src/time.cr]: A simpler implementation and more explicit: `leap_year?(year) ? 366 : 365`

Comment (coderhs) [reply src/time.cr]: I agree and also faster. I will modify the code for that.

File: src/time.cr (hunk 1 above)
Comment (coderhs) [reply src/time.cr]: @luislavena thanks for the link. I wasn't aware of that, changing the code to such format.

PS: There are similar methods without such code examples, would a PR with such doc changes be welcomed 😄  would love to contribute like that.

**Comment by coderhs** [comment]
@luislavena & @straight-shoota PR has been updated.

**Comment by luislavena** [comment]
>  There are similar methods without such code examples, would a PR with such doc changes be welcomed 😄 would love to contribute like that.

Definitely @coderhs!!! All the contributions that introduce better and easier to use documentation are welcome! 😄 

Thank you!
❤️ ❤️ ❤️

**Comment by coderhs** [comment]
@luislavena have added code example to `days_in_month` and a test for that, have added the test for that method along with this PR, as mentioned in issue #5157

**Pull request merged**